.gitmodules

```diff
@@ -184,3 +184,6 @@
 [submodule "third_party/mbed-mcu-boot/repo"]
 	path = third_party/mbed-mcu-boot/repo
 	url = https://github.com/ATmobica/mcuboot.git
+[submodule "third_party/webos_sdk"]
+	path = third_party/webos_sdk
+	url = https://github.com/cabin15/webos-ose-ndk
```

Comment:
You use your own account @cabin15  in the github link path , no one has a more official repo (i guess for LG)
that fine but you know i would not recommend that for long term solution

Reply:
@jmeg-sfy 
You are right.
Our team is about on the start line about matter work and contribution.
It would be so kind if you could think this PR as a start baseline.
I'll drive internal discussion for providing webOS SDK as LG official repo.
Thank you for your review.